lib/src/const.nr

```diff
@@ -1,13 +1,17 @@
 /// Node type
 global BRANCH: u64 = 2; global EXTENSION: u64 = 0; global LEAF: u64 = 1;
 
-/// Constants for 32-byte long keys; TODO: Omit these when it is possible to replace constants with numeric generics.
-global KEY_LENGTH: u64 = 32; // (fixed) key length in bytes
-global NIBBLE_LENGTH: u64 = 64; // = 2*KEY_LENGTH
+/// Length of keccak256 hash.
+global HASH_LENGTH: u64 = 32;
+global HASH_NIBBLE_LENGTH: u64 = 64; // = 2 * HASH_LENGTH
```

Comment:
It seems we are fixing formatting here, (add missing spaces), I would separate that to another PR, make it easier to review for lib author.